packages/fast-components-react-base/src/divider/README.md

```diff
@@ -1,2 +1,5 @@
 # Divider
-The *Divider* component can be an [hr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) element.
+The *divider* component defines a thematic break or shift of topic within a page. It is an enhancement of the HTML [hr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) element.
```

Comment:
Is this actually an enhancement of the `<hr />` element? It essentially returns an <hr /> and ensures that we handle a11y correctly. I don't see it as enhancing the element in any meaningful way